app/models/qernel/initializer_inputs.rb

```diff
@@ -0,0 +1,19 @@
+module Qernel
+  class InitializerInputs
+    def self.for_area(area_code)
+      new(area_code).all
+    end
+
+    def initialize(area_code)
+      @area_attributes =
+        Etsource::Loader.instance.area_attributes(area_code)[:init] || {}
```

Comment:
`Etsource::Loader#area_attributes()` will directly access the `Atlas::Dataset`, although the dataset is already loaded in `graph.dataset`. Hence, I strongly believe the loading of the init. inputs for a particular derived dataset should remain in graph.rb (or area.rb).